src/dagmc/DagMC.cpp

```diff
@@ -870,11 +870,11 @@ ErrorCode DagMC::next_vol(EntityHandle surface, EntityHandle old_volume,
 
 /* SECTION III: Indexing & Cross-referencing */
 
-EntityHandle DagMC::entity_by_id(int dimension, int id) const {
+EntityHandle DagMC::entity_by_id(int dimension, int id) {
```

Comment:
this is a regression from another PR that added the `const` - please remove this change